### PR TITLE
Add reroute processor to stream a log page

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
This PR closes [Issue 3125](https://github.com/elastic/observability-docs/issues/3125) and adds a section on using the reroute processor to simplify log data management.

### CODE REVIEW

File: docs/en/observability/logs-stream.asciidoc [hunk 1]
```diff
@@ -855,4 +870,141 @@ PUT /logs-example-default/_mapping
...
+  "description": "Extracts fields and reroutes WARN",
+  "processors": [
+    {
+      "dissect": {
+        "field": "message",
+        "pattern": "%{@timestamp} %{log.level} %{host.ip} %{message}"
+      },
+      "reroute": {
+        "tag": "high_severity_logs",
+        "if" : "ctx.log?.level == 'WARN' || ctx.log?.level == 'ERROR'",
```
Comment (ruflin) [inline docs/en/observability/logs-stream.asciidoc:918]: @felixbarny Even though the example is correct not sure  if we should encourage to route on severity levels for logs? I rather use an example with service.name?

**Comment by ruflin** [comment]
@felixbarny Would be great to get your review on this one.

File: docs/en/observability/logs-stream.asciidoc (hunk 1 above)
Comment (felixbarny) [reply docs/en/observability/logs-stream.asciidoc:918]: I don't think that's a bad example to be honest. One thing I would add is to add a recommendation to keep the number of data streams relatively low. We can link to this page for more details: https://www.elastic.co/guide/en/elasticsearch/reference/current/size-your-shards.html.

Also, we should probably advocate for using the field API here

```suggestion
        "if" : "$('log.level', '') == 'WARN' || $('log.level', '') == 'ERROR'",
```

**Review by felixbarny — APPROVED** [review]

File: docs/en/observability/logs-stream.asciidoc (hunk 1 above)
Comment (felixbarny) [reply docs/en/observability/logs-stream.asciidoc]: @mdbirnstiehl have you tested that this syntax works? I've just added that suggestion from how I remember the syntax.

I'd also like to see a follow up PR that links to https://www.elastic.co/guide/en/elasticsearch/reference/current/size-your-shards.html.

Comment (mdbirnstiehl) [reply docs/en/observability/logs-stream.asciidoc]: I'll verify the syntax and update accordingly. I added a note on line 893 linking to the size your shards doc

**Pull request merged**